Description: Upstream changes introduced in version 2010.09~rc1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 u-boot (2010.09~rc1-1) unstable; urgency=low
 .
   * New upstream release candidate.
     - Drop guruplug-miiphy_reset.diff.
     - Update mipsel-native-endianness.diff.
     - Drop sh4-native-compile.diff.
   * Bump to Standards-Version 3.9.1.
 .
 The person named in the Author field signed this changelog entry.
Author: Clint Adams <schizo@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- u-boot-2010.09~rc1.orig/snapshot.commit
+++ u-boot-2010.09~rc1/snapshot.commit
@@ -1 +1 @@
-2d941de9d5c7ba00dc19787dfa0aac2949fd00fb  Fri, 10 Sep 2010 00:16:19 +0200
+$Format:%H  %cD$
